pontoon/homepage/views.py

```diff
@@ -28,6 +30,23 @@ def homepage(request):
     else:
         start_url = reverse('pontoon.teams')
 
-    return render(request, 'homepage.html', {
+    playground_url = reverse('pontoon.translate.locale.agnostic', kwargs={
+        'slug': 'tutorial',
+        'part': 'playground',
+    })
+
+    intro_url = reverse('pontoon.translate.locale.agnostic', kwargs={
+        'slug': 'pontoon-intro',
+        'part': 'messages.properties',
+    })
+
+    homepage = Homepage.objects.first()
```

Comment:
You mean `.last()`?

Reply:
Since we would like to display the most recent iteration, so shouldn't it be first, considering we used "-created_at" ordering in the models?

Reply:
I thought you'd remove that:
https://github.com/mozilla/pontoon/pull/1206/files#r257930665